data/ships.txt

```diff
@@ -1480,6 +1480,46 @@ ship "Gunboat"
 
 
 
+ship "Harvester"
+	sprite "ship/mining drone"
+	attributes
+		category "Drone"
+		"cost" 100000
+		"shields" 200
```

Comment:
The military human drones don't have shields. I find it odd that a civilian drone would.